File: TugasPraktikum06/Bahasa Python/PRAK605-2310817320013-AikoAnatashaWendiono.py
def matriks(A, B):
    n = len(A)
    hasil = [[0] * n for _ in range(n)]
    
    for i in range(n):
        for j in range(n):
            for k in range(n):
                hasil[i][j] += A[i][k] * B[k][j]
    
    return hasil

def display_matrix(matrix):
    for row in matrix:
        print(" ".join(map(str, row)))

def main():
    n = int(input())
    
    print("Matriks A")
    A = [list(map(int, input().split())) for _ in range(n)]
    
    print("Matriks B")
    B = [list(map(int, input().split())) for _ in range(n)]
    
    hasil = matriks(A, B)
    
    print("Matriks AXB")
    display_matrix(hasil)

if __name__ == "__main__":
    main()

File: TugasPraktikum06/Bahasa C/PRAK605-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

void perkalian_matriks(int A[100][100], int B[100][100], int hasil[100][100], int n) {
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            hasil[i][j] = 0;
            for (int k = 0; k < n; ++k) {
                hasil[i][j] += A[i][k] * B[k][j];
            }
        }
    }
}

void display_matriks(int matrix[100][100], int n) {
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            printf("%d", matrix[i][j]);
            if (j < n - 1) {
                printf(" ");
            }
        }
        printf("\n");
    }
}

int main() {
    int x;

    scanf("%d", &x);

    int A[100][100], B[100][100], hasil[100][100];

    printf("Matriks A\n");
    for (int i = 0; i < x; ++i) {
        for (int j = 0; j < x; ++j) {
            scanf("%d", &A[i][j]);
        }
    }

    printf("Matriks B\n");
    for (int i = 0; i < x; ++i) {
        for (int j = 0; j < x; ++j) {
            scanf("%d", &B[i][j]);
        }
    }

    perkalian_matriks(A, B, hasil, x);

    printf("Matriks AXB\n");
    display_matriks(hasil, x);

    return 0;
}
